### PR TITLE
feat(search): display selected saved searches as shortcuts

### PR DESCRIPTION
**Pull request by petrjasek**
SDANSA-196

requires https://github.com/superdesk/superdesk-core/pull/1577

### CODE REVIEW

File: scripts/apps/search/controllers/SearchMenuController.ts [hunk 1]
```diff
@@ -1,26 +1,61 @@
+import {get, isEqual} from 'lodash';
+import {ISavedSearch} from '../SavedSearch';
```
Comment (tomaskikutis) [inline scripts/apps/search/controllers/SearchMenuController.ts]: What does the dropdown in the search page contain now? It used to only contain a provider e.g. a source a search to be executed on. Does the dropdown now contain both providers and shortcuts? Do shortcuts work with any provider? I would split them into groups so it's easy to know which ones are providers/shortcuts.

File: scripts/apps/search/views/save-search-dialog.html [hunk 2]
```diff
@@ -13,5 +13,9 @@
             <span id="search_global" sd-switch ng-model="edit.is_global"></span>
             <label class="label--inline" translate>Make global</label>
         </div>
+        <div class="field" ng-if="edit.is_global">
```
Comment (tomaskikutis) [inline scripts/apps/search/views/save-search-dialog.html]: Doesn't this need the same feature flag check as in https://github.com/superdesk/superdesk-client-core/pull/2925/files#diff-275cb0eb9ee87557fa6c69e095e061deR85 ?

**Comment by tomaskikutis** [comment]
Could you explain how bundling shortcuts with its repo works?

**Comment by petrjasek** [comment]
it's like:
- superdesk
- superdesk shortcut 1
- superdesk shortcut 2
- external source
- external source shortcut 1
- external source shortuct 2

**Pull request merged**